tests/api/console/test_cli_console_output.py

```diff
@@ -74,11 +73,19 @@ def test_error_messages(cli_console, capsys):
     assert_output_matches("42\n  73\n  ops\nOPS\n", capsys)
 
 
-def test_phase_nesting_not_allowed(cli_console):
+def test_phase_nesting(cli_console, capsys):
```

Comment:
step nesting isn't allowed still, right? We should capture that in a test since it's part of the contract.

Reply:
Similar question to https://github.com/snowflakedb/snowflake-cli/pull/1843#discussion_r1838759925, I'm not entirely sure what you mean by step nesting, since a step is just a message and cannot contain things within it

Reply:
as you pointed out, steps are just messages, not blocks like phases, so this comment doesn't make sense. My bad.